tgui/packages/tgui/interfaces/BorgHypo.js

```diff
@@ -0,0 +1,74 @@
+import { toFixed } from 'common/math';
+import { useBackend } from '../backend';
+import { Button, Flex, NoticeBox, Section, ProgressBar } from '../components';
+import { Window } from '../layouts';
+
+export const BorgHypo = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    maxVolume,
+    theme,
+    reagents,
+    selectedReagent,
+  } = data;
+  return (
+    <Window
+      width={450}
+      height={350}
+      theme={theme}
+    >
+      <Window.Content scrollable>
+        <Section>
+          <Reagent
+            reagents={reagents}
+            selected={selectedReagent}
+            maxVolume={maxVolume} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Reagent = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { reagents, selected, maxVolume } = props;
+  if (!reagents) {
+    return (
+      <NoticeBox>
+        No reagents available!
+      </NoticeBox>
+    );
+  }
+  return reagents.map(reagent => (
+    <Flex key={reagent.name} mb={1}>
```

Comment:
I have refactored this thing a bit. You did not need an extra if inside, as long as there is no extra `0` in the list, that you inserted in ui_data in DM.

By removing the if, this simplifies by a lot, and triggers an eslint warning, which is actually a valid concern, because if you don't specify a key, there could be visual bugs with this list. I inserted the key for you.

I also reduced all gaps to 1 unit.

Please check if it works after all edits and feedback.